sys-boot/refind/refind-0.11.4.ebuild

```diff
@@ -0,0 +1,168 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic multiprocessing toolchain-funcs
+
```

Comment:
why `flag-o-matic` and `multiprocessing`?

Reply:
`multiprocessing` is used for `makeopts_jobs`, `flag-o-matic` was used in previous version and is now useless. I don't know how or even if it is possible, but it would be good to have `repoman` detect those cases…